src/cryptoadvance/specter/server_endpoints/settings.py

```diff
@@ -45,6 +45,9 @@ def bitcoin_core():
     if not current_user.is_admin:
         flash("Only an admin is allowed to access this page.", "error")
         return redirect("")
+    # The node might have been down but is now up again
+    # (and the checker did not realized yet) and the user clicked "Configure Node"
+    app.specter.check()
```

Comment:
Is this needed even if the RPC is set? I know it's good to check anyway but this means potentially significant slowdown for loading the settings page. So if this is needed only if Specter thinks no node is connected maybe we should check if RPC is None and only if it is do it?

Reply:
Ok, i've added a check of app.specter.rpc being None beofre doing this.